src/scripts/main.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, () => {
+      resolve(`
+        It was ${eventName} on the element: ${element.nodeName},
+        id: ${element.id}.
+      `);
+    });
+  });
 }
 
-const printMessage = (message) => {
-// write your code here
+function printMessage(message) {
+  document.body.insertAdjacentHTML('beforeend', `
+  <div class="message">${message}</div>
+  `);
 };
```

Comment:
```suggestion
function printMessage(message) {
  document.body.insertAdjacentHTML(
    'beforeend', 
    `<div class="message">${message}</div>`
  );
};
```
I think it's clearer that way